server/controllers/user/user.coffee

```diff
@@ -1,3 +1,12 @@
+# The 'user' endpoint is dedicated to opeartions on the user
+# by the user herself once authentified.
+
+# Other controllers involving operations on users:
+# - auth: authentification operations (signup, login, etc)
+# - relations: handling friends relations
+# - invitatitons: inviting people out of Inventaire
+# - users: finding users by their usernames, positions, etc
+
```

Comment:
yeeeeeesss more like that more more !

Reply:
:D

Reply:
I needed it myself to understand why `invitations` should stay a standalone endpoint and not being incorporated into one of the other existing ones